src/admin/dashboard/pages/modules/UpgradeModal.tsx

```diff
@@ -14,341 +13,242 @@ const UpgradeModal = ( {
 
     return (
         <Modal
-            className="max-w-sm"
+            className="dokan-upgrade-modal max-w-[720px] mt-10 bg-transparent"
             isOpen={ isOpen }
-            showXButton={ true }
+            showXButton={ false }
```

Comment:
We should not use class `dokan-upgrade-modal` and  why are we using image as **x** button instead of `showXButton` option?

Reply:
Bhai, we needed a custom close button based on the mockup. But the Modal component doesn't support custom button styling. So, we created our own button by disabling the default one.